tensorflow-ops/src/TensorFlow/Gradient.hs

```diff
@@ -813,6 +813,10 @@ opGrad "Placeholder" _ _ _ = []
 opGrad "VarHandleOp" _ _ _ = []
 opGrad "Variable" _ _ _ = []
 
+opGrad "Sqrt" _ [toT -> x] [dz] = [Just $ sq' `CoreOps.mul` dz]
+  where
+    sq' = vector [1] `CoreOps.div` (vector [2] `CoreOps.mul` CoreOps.sqrt x)
```

Comment:
I don't remember, is there a way to multiply by a scalar rather than a (implicitly?) replicated vector?

Reply:
I think

```haskell
scalar 1 `CoreOps.div` (scalar 2 `CoreOps.mul` CoreOps.sqrt x)
```

would work,  but if that doesn't compile, then the proposed implementation is fine.

Reply:
Yes, the multiplication with a scalar works. It compiles.